### DISCUSSION TITLE
Paging Bug

### DISCUSSION THREAD

**Discussion by hamzaazman** [community/community#63754] [Mobile] [answered]
### Select Topic Area

Bug

### Body

When scrolling through the repository, if I enter and then exit a repository, I am directed back to the top of the list.

https://github.com/community/community/assets/60855026/46c335c4-8110-4964-8b5c-6e16a9f30c5b

**Comment by eliperkins** [comment, marked as answer]
Hey @hamzaazman , thanks for this feedback!

This does look like a bug to me too, so I've filed an issue in our internal tracking repo to get this fixed. I'll work with our engineering team and let you know when the bug is fixed.

Thanks for helping improve GitHub Mobile!
